src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -209,26 +203,68 @@ public void run() {
       view.variable(context.getNamer().getSampleResponseVarName(context));
 
       if (context.getMethodConfig().getPageStreaming() != null) {
-        type =
-            context
-                .getMethodConfig()
-                .getPageStreaming()
-                .getResourcesFieldConfig()
-                .getField()
-                .getType()
-                .makeOptional();
+        FieldConfig resourceFieldConfig =
```

Comment:
I think I know where this is going but the flow control is scary (I'm having a hard time matching braces). I think you can flatten this.

Something like
```java
boolean pageStreaming = context.getMethodConfig().getPageStreaming() != null;
boolean pageStreamingResourceField = ....;


if (pageStreamingResourceField) {
  // ...
  return
}
if (pageStreaming) {
  // ...
} else {
  // ...
}
```